airbyte-webapp/src/components/JobItem/components/AttemptDetails.tsx

```diff
@@ -5,20 +5,20 @@ import { FormattedMessage, useIntl } from "react-intl";
 
 import Status from "core/statuses";
 
-import { AttemptRead, JobConfigType } from "../../../core/request/AirbyteClient";
+import { AttemptRead } from "../../../core/request/AirbyteClient";
 import styles from "./AttemptDetails.module.scss";
 
-interface IProps {
+interface AttemptDetailsProps {
   className?: string;
   attempt: AttemptRead;
-  configType?: JobConfigType;
```

Comment:
:information_source: This property wasn't used anywhere inside the component so I removed it.